.github/workflows/ci.yml

```
@@ -17,7 +17,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                python: [39, 310, 311, 312, 313]
+                python: [39, 310, 311, 312, 313, '3.13t']
```

```suggestion
                python: [39, 310, 311, 312, 313, 313t]
```
(matching convention used below)

---

I wasn't entirely sure if the uv action would recognize it, just playing it safe. The format for cibuildwheel comes from their docs. I'll give it a try.